### PR TITLE
[W4.11d][W16-4]LAW YONG QIANG CLINTON

### PR DESCRIPTION
**Pull request by clawyq**

### CODE REVIEW

File: src/seedu/addressbook/common/Utils.java [hunk 1]
```diff
@@ -13,6 +13,7 @@
      * Returns true if any of the given items are null.
      */
     public static boolean isAnyNull(Object... items) {
+        if (items == null) return true;
```
Comment (bqnguyen94) [inline src/seedu/addressbook/common/Utils.java:16]: Can you tell me what this line does and why you use it? :)

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -43,4 +44,23 @@ private void assertAreUnique(Object... objects) {
...
+
+    @Test
+    public void isAnyNullTest() throws Exception {
+        boolean result1 = isAnyNull(1, 2, 3, 4);
+        boolean result2 = isAnyNull("a", 'b', "c");
+        boolean result3 = isAnyNull("a", 'b', 1, 2);
+        boolean result4 = isAnyNull("a", "bc", 1, null ,2);
+        boolean result5 = isAnyNull(null);
+        boolean result6 = isAnyNull(1, null);
+        boolean result7 = isAnyNull(null, null);
```
Comment (bqnguyen94) [inline test/java/seedu/addressbook/common/UtilsTest.java:56]: Also, I think these booleans are not really necessary.

File: src/seedu/addressbook/common/Utils.java (hunk 1 above)
Comment (clawyq) [reply src/seedu/addressbook/common/Utils.java:16]: To avoid an error if isAnyNull is called on just a single null.

Comment (bqnguyen94) [reply src/seedu/addressbook/common/Utils.java:16]: 👍 You can close this PR now

**Pull request closed without merging**